force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +510,18 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        // Compare the ages of the two Person objects
+    if (person1.getAge() > person2.getAge()) {
+        // If the first Person is older, return the first Person
+        return person1;
+    } 
+    else if (person2.getAge() > person1.getAge()) {
```

Comment:
Your second check is probably redundant. There are only 2 choices, so if it does not meet your first check, it would have to be the second option.